tldraw/tldraw
# Add migration for horizontal alignment

This adds a migration to migrate existing alignment options to their legacy counter parts (`start` -> `start-legacy`, `end` -> `end-legacy`, `middle` -> `middle-legacy`).

With this change the legacy options don't show any align as active in the Styles panel:
![CleanShot 2023-05-23 at 19 53 45](https://github.com/tldraw/tldraw/assets/2523721/4017e03a-9492-4a02-b991-ac206f40ae17)

I think this is probably what we want.

### Change Type


### Test Plan

1. Use some old preview link to create Geo and Note shapes with old alignment options. You can use this one:
https://examples-kzwtf68jr-tldraw.vercel.app/
2. Copy and paste these shapes over to staging. Nothing should change visually.
3. Also try out exporting to svg (with both old and new alignment options)


### Release Notes

- Add support for legacy alignment options.
